animalfacts.py

```diff
@@ -2866,6 +2867,23 @@ def animalfactsbot(reddit):
     'The saliva of a vampire bat can be used to prevent blood from clotting.',
     )
 
+VULTURE_FACTS = (
+    'Vultures eat animals that have died in the wild. Without them, these animals would rot and smell.',
+    'Vultures heads and necks are almost bare so they stay clean while feasting on rotten meat.',
+    'Vultures also have strong immune systems so they don’t get sick from eating rotten meat.',
+    'Vultures have wide, strong wings. They can glide in the air for hours looking for a meal.',
+    'Vultures eat as much as they can at one meal. They never know when the next meal will come.',
+    'Vultures live in every part of the world except Australia and the Antarctica.',
+    'To eat their prey vultures have sharp hooked beaks and talons. They can also use tools.',
+    'Vultures sometimes drop eggs to break them or hit them against rocks to open them for food.',
+    'Vultures often appear when an animal is dying or dead. Egyptians and Native Americans used vultures in burial ceremonies.',
+    'Unlike many raptors, vultures are relatively social and often feed, fly or roost in large flocks. ',
+    'Vultures have excellent senses of sight and smell to help them locate food, and they can find a dead animal from a mile or more away.',
+    'While vultures eat mostly dead animals, they are capable of attacking and will often prey on extremely sick, wounded or infirm prey.',
+    'Vultures urinate on their legs and feet to cool off on hot days, a process called urohydrosis.',
+    'When threatened, vultures vomit to lighten their body weight so they can escape more easily into flight. '
```

Comment:
Could you pop a quick comma at the end of this line to match the other blocks? Otherwise these are all great facts!

Reply:
No problem.
Yeah, they are !!!